### ISSUE TITLE
Serious accessibility issues

### ISSUE THREAD

**Issue by vlajna95** [ubik23/charactercreator#194]
As a blind person, I'm using a screen-reader. The options in the menus really lack accessibility. 
Since I suppose they are based on images, I'd suggest adding alt attributes to them. That would really enhance the usability of the website for a blind or visually impaired person and requires just a simple edit.

**Comment by ubik23** [comment]
Hello Vlajna95, your concerns are valid though I cannot implement them all. The options in the menu is an easy fix to do but the images displayed are partial and in the SVG file format which doesn't allow for simple alt attributes. 
As for the colors, I think we can do that.

**Comment by vlajna95** [comment]
Great, something is better than nothing. :) 
I'll research about labeling SVGs, maybe there is something doable.
